catalogue/webapp/pages/concept.tsx

```diff
@@ -393,17 +447,6 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
       imagesByPromise,
     ]);
 
-    if (conceptResponse.type === 'Error') {
-      if (conceptResponse.httpStatus === 404) {
-        return { notFound: true };
-      }
-      return appError(
-        context,
-        conceptResponse.httpStatus,
-        conceptResponse.description
-      );
-    }
```

Comment:
This is an unrelated improvement I spotted while reading this code – if the concept lookup fails, there's no point waiting to see if we can get related works/images; we might as well fail immediately.